2_form.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My first form here</title>
+</head>
+<body>
+
+  <h1>Payment Form</h1>
+  <h4>Requierd Fields are followed by .*.</h4>
+  <p style="font-size:30px">Contact Information</p> <!--Using this to write here in large size-->
+  <fieldset> 
+   <legend>Contact Information</legend>  
+  <form action="" method="GET"> 
+  <fieldset>
+  <legend>Choose</legend>
+<div style="white-space: pre">
+  <label>Are you Teacher or Student ?</label> <!--Correction done-->
+
+<label for="type">Student</label> <input type="radio" id="type" name="radiotype" value="Student">
+ <!--Adding empty lines to make it spacing between them-->    
+<label for="teachertype">Teacher</label> <input type="radio" id="teachertype" name="radiotype" value="Teacher">
+</div> 
+</fieldset>
+
+<div style="white-space: pre">
+ <label for="">Name .*.         </label> <input type="text" name="username" id="" placeholder="Username" required>    
```

Comment:
for and id attributes are empty

Reply:
Mam you said to leave it empty if there is no use of it?

Reply:
But in the example you indeed using label Name with input username. So in order to connect them we use for in label and id in input. you have them but for some reasone you didnt out any value. So please add the value.